Tourist-Places/MrRobot-Hyderabad.md

```diff
@@ -0,0 +1,11 @@
+# Tourist Places
+
+- Charminar
+- Tank bund
+- IMAX
+- Golconda Port
+
+# Food
+
+- Chicken Biryani
```

Comment:
Good to hear that.! Yeah! Hyderabad is famous for Chicken Biryani! Hope you enjoyed !